revised/stellaswap-proposal

```diff
@@ -0,0 +1,237 @@
+
+With a TVL of more than 12X, USDC-GLMR has a higher slippage when trading a value of $5k. Although this clearly highlights the power of Pulsar, the emphasis should be on tracking the desirable price impact degree that ecosystem grants will foster.
+
+**2) Trading Volume**
+
+Greater volume indicates greater user activity on the chain, and is perhaps one of the most important metrics to optimize for a DEX. We were successful in attracting volume to the ecosystem as StellaSwap became the first DEX in all of Polkadot to surpass $1B in cumulative trading volume. Even with a lower grant amount, we’re confident of attracting a higher pace of trading volume when Pulsar is live with ecosystem rewards. Our average volume of around $2M daily was within the targets of the previous grants program, which we hope to surpass with the utilization of the upcoming grants.
+
+**3) Total Value Locked (TVL)**
+
+It is inevitable that TVL is regarded as one of the strongest indicators of popularity and health of any ecosystem. The higher the TVL, the stronger the ecosystem as it is more trusted and reliable. As we’ve hit the TVL targets that were previously forecasted, TVL will continue to be an important - but not defining - metric of evaluation. This is because TVL is quoted in USD, and is affected by numerous macro factors that may affect TVL targets which are beyond our control. Systemic shocks, volatility of bluechips and network effects can all affect our TVL targets.
+
+The TVL forecast is highlighted in the *Use of Grants* section.
+
+**4) Unique Users**
+
+User acquisition is a core metric for growth of any protocol; a growing protocol is attributed - in part - to an increase in the number of users. It is imperative to track and optimize for user acquisition, especially when ecosystem incentives are used. The objective is to attract and onboard as many users as we can onto the Moonbeam network.
+
+**USE OF GRANTS TIMELINE**
+
+*It must be emphasized that the totality of the 2M grants will be used to incentivize farms on both Pulsar & stable AMM to deepen liquidity. Under no circumstances would any of the grants be used for development or any other purposes beyond incentivizing the farms, which goes to LP stakers.*
+
+This current round of grants will run for approximately 3 months, and will be used to incentivize strategic farms as part of the transition to Pulsar. Strategic farms are made up of assets of strategic importance, which are in the form of blue-chips. Pulsar will ensure that grants will go a longer way in attracting more sticky and sustainable liquidity. Although the TVL forecast of this current grants round would be similar to our previous grant application, the market depth from Pulsar’s TVL will be multiplied given the greater concentration of liquidity. This will allow protocols within the network to tap on our deep liquidity for their full-functioning.
+
+**![|602x115](upload://zaPvQlJ43fRBjv45RXAJh6LgxrI.png)**
```

Comment:
Can you add the image to the PR - otherwise this will show a blank image placeholder.